sms_sponsorship/webapp/src/i18n.js

```diff
@@ -164,6 +164,7 @@ i18n
                     "waitingForChild": "Bitte warte ein paar Sekunden, wir suchen ein Kind, das auf einen Paten wartet ...",
                     "waitingForOtherChild": "Wir suchen ein Kind, das zu dir passt, bitte warte ein paar Sekunden ...",
                     "ageYears": " Jahre",
+                    "notMatchCriteria": "Could not found a child with given criteria.",
```

Comment:
Could you enter the translations please? Deepl should work fine for that (maybe check German with Andre).